.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.0.3
+current_version = 0.0.4
```

Comment:
Please can you roll back this version number - that should be changed as part of the release process.